videosink/handler.go

```diff
@@ -23,6 +25,17 @@ var bufferPool = sync.Pool{
 	},
 }
 
+// stopAndDrain ensures that the given timer is stopped and has no pending
+// event.
+func stopAndDrain(timer *time.Timer) {
+	if !(timer == nil || timer.Stop()) {
```

Comment:
I feel this is more readable: `if timer != nil && !timer.Stop() {`

Reply:
Changed even though my personal preference is to avoid chains with multiple negations.